src/include/platform/CHIPDeviceConfig.h

```diff
@@ -617,6 +617,22 @@
 #define CHIP_DEVICE_CONFIG_BLE_ADVERTISING_INTERVAL_CHANGE_TIME 30000
 #endif
 
+#ifndef CHIP_DEVICE_CONFIG_BLE_EXT_ADVERTISING
+#define CHIP_DEVICE_CONFIG_BLE_EXT_ADVERTISING 0
+#endif
+
+#if CHIP_DEVICE_CONFIG_BLE_EXT_ADVERTISING
+
+#ifndef CHIP_DEVICE_CONFIG_BLE_EXT_ADVERTISING_INTERVAL_CHANGE_TIME
+#define CHIP_DEVICE_CONFIG_BLE_EXT_ADVERTISING_INTERVAL_CHANGE_TIME (15 * 60 * 1000)
+#endif
+
+#ifndef CHIP_DEVICE_CONFIG_BLE_EXT_ADVERTISING_INTERVAL
+#define CHIP_DEVICE_CONFIG_BLE_EXT_ADVERTISING_INTERVAL 1920
+#endif
```

Comment:
Please document all of these constants

Reply:
what is this interval measured in? please add suffix.

Reply:
@andy31415  Added to the comments documentation same as all the others. Again, I would be happy to change the define as well but then we should probably adjust all of them in that file, at least the BLE ones for consistency, not to confuse new users.